packages/dynamite/dynamite/lib/src/type_result/type_result.dart

```diff
@@ -46,17 +46,42 @@ abstract class TypeResult {
 
   String? get _builderFactory;
 
-  String serialize(final String object);
+  /// Serializes the variable named [object].
+  ///
+  /// The serialized result is an [Object]?
+  String serialize(final String object) => 'jsonSerializers.serialize($object, specifiedType: const $fullType)';
 
-  String deserialize(final String object, {final bool toBuilder = false});
+  /// Deserializes the variable named [object].
+  ///
+  /// The serialized result will be of [name].
+  String deserialize(final String object) =>
```

Comment:
Ideally I'd like to call decode automatically just like encode does with serialize.